src/js/common/helpers/login-helpers.js

```diff
@@ -39,6 +39,8 @@ export function getUserData(dispatch) {
       dispatch(updateProfileField('status', json.data.attributes.va_profile.status));
       dispatch(updateProfileField('services', json.data.attributes.services));
       dispatch(updateLoggedInStatus(true));
+    } else {
+      dispatch(profileLoadingFinished());
```

Comment:
@annekainicUSDS It looks like the else is hit whenever there's a non-200 server response. We get json back from the server, but it has no data attribute.

That's sort of brittle; we should probably be checking `reponse.ok` and acting accordingly.